src/main/java/org/openstreetmap/atlas/geography/atlas/change/ChangeAtlas.java

```diff
@@ -62,6 +66,49 @@ public ChangeAtlas(final Atlas source, final Change... changes)
         new AtlasValidator(this).validate();
     }
 
+    public ChangeAtlas(final Change... changes)
+    {
+        if (changes == null || changes.length < 1)
+        {
+            throw new CoreException("Change cannot be null in a ChangeAtlas.");
```

Comment:
Nit: "Change cannot be null or empty in a ChangeAtlas."
This or consider breaking this out into two different checks, so that the error message is specific and clear. It will probably help with debugging in the future.